docs/README.md

```diff
@@ -0,0 +1,50 @@
+[KroxyDoc]: https://kroxylicious.io/ "Kroxylicious documentation"
+[AsciiDoc]: https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/  "AsciiDoc reference"
+
+<!-- omit from toc -->
+# Kroxylicious documentation
+
+Welcome to the Kroxylicious documentation! 
+
+This folder contains the source files for managing the Kroxylicious documentation. The documentation is written in [AsciiDoc][AsciiDoc] and generated using 
+ and it provides content to help you understand and set up Kroxylicious proxies.
+
+<!-- omit from toc -->
+## Table of Contents
```

Comment:
Perhaps the `Rendering documentation` section from the dev guide should be moved to this file and the DEV_GUIDE should link to this README.